content/community/getting-involved/promotion/promotion-team.md

```diff
@@ -0,0 +1,72 @@
+
+Apart from Auckland University, mmu_man, a member of the Haiku Community, also published a [research paper about the interoperability of filesystem attributes](https://dcpapers.dublincore.org/pubs/article/view/3633/1859), as well as a [research paper on using QEMU for OS development](http://adt.cs.upb.de/quf/quf11/QUF11-papers/quf2011-11.pdf). mmu_man also made a [poster advertising the benefits of using Haiku for research](http://eurosys2010.sigops-france.fr/poster_demo/eurosys2010-final17.pdf). [Colin Günter did his master thesis on Haiku's WLAN stack](https://www.haiku-os.org/news/2016-02-26_wlan_master_thesis_published/), whilst Rudolf Cornelissen wrote a [thesis on writing video card drivers](https://www.haiku-os.org/legacy-docs/writing-video-card-drivers/).     
+We aim to attract more research groups by promoting the uses of Haiku for research, and we also want to reach out to Auckland University and see if their UI research group is interested in coming back to Haiku to conduct any future research.     
+
+Additionally, we have participated in [Google Summer of Code](https://summerofcode.withgoogle.com/) and [Google Code-In](https://codein.withgoogle.com/) (when it was still operating), providing opportunities for students to contribute to our project and develop their coding skills. We also participated in the [Semester of Code](http://semesterofcode.com/) but unfortunately we didn't receive any student applications. 
+
+#### Businesses
+Businesses can benefit from Haiku as an operating system, and Haiku has the potential to be a lightweight alternative to the proprietary desktop operating systems businesses currently use. An information sheet for businesses has been drafted, which gives businesses a rundown of Haiku and its advantages and disadvantages as a business-oriented operating system. However, to attract more businesses we must improve Haiku further, and hence we will delay marketing efforts for businesses until Haiku reaches a stable and mature stage. 
+
+#### Partnerships with other Haiku-related sources
+If you haven't heard of them already, [BeSly](https://www.besly.de/index.php/de/) and [Haikunauts](https://haikunauts.home.blog/) are the two major third party Haiku sources. BeSly is a wiki with tips and tricks on using Haiku, whilst Haikunauts is a blog aimed at promoting Haiku as an OS. As mentioned above, we will work in partnership with these two sources and see what we can do together to help spread the word about Haiku.     
```

Comment:
`the two major third party Haiku sources` -> `resources about Haiku managed by fans of the project.`